tattoo.ts

```diff
@@ -79,16 +80,49 @@ const cli = cliArgs([
         "Set to clone all repos from remote instead of updating local copies."
   },
   {
+    name: "released",
+    type: Boolean,
+    defaultValue: false,
+    description:
+        "Set to update repos to the latest release when possible."
+  },
+  {
+    name: "branchconfig",
+    type: String,
+    defaultValue: "",
+    description:
+        "Set to use a config file to override branches/orgs for particular repos."
```

Comment:
Link to docs on the branch config doc? Also, why not have a default value so that if tattoo is run from a directory it just picks up the config.